spec/lang/step/intrinsics.md

```diff
@@ -383,3 +383,69 @@ impl<M: Memory> Machine<M> {
     }
 }
 ```
+
+We also implement the atomic fetch operations. First we define a helper function to decide which operations can be in a fetch operation.
```

Comment:
I think this can be just part of the `Atomic accesses` block.

Reply:
Should I put the helper function (if we will still use it) at the beginning of this block or keep it in the middle?

Reply:
You can put the helper function inside the intrinsic function. :)

Though I think I'd prefer to have this as part of well-formed. Making it UB to pass the wrong operand doesn't feel right.